flow/flow.go

```diff
@@ -150,10 +138,21 @@ func FLowsFromSFlowSample(sample *layers.SFlowFlowSample, probePath *string) []*
 		/* FIX(safchain): just keeping the raw packet for now */
 		record, ok := rec.(layers.SFlowRawPacketFlowRecord)
 		if !ok {
+			logging.GetLogger().Critical("1st layer is not SFlowRawPacketFlowRecord type")
 			continue
 		}
 
-		flow := New(&record.Header, probePath)
+		packet := &record.Header
+		key := (FlowKey{}).fillFromGoPacket(packet)
+		flow, new := ft.GetFlow(key.String(), packet)
+		if new {
+			flow.ProbeGraphPath = *probePath
+
+			flow.DebugTimestamp = time.Now().Unix()
```

Comment:
What is the purpose of these fields ? If this is only for debugging purpose this should be removed, if this is for something else maybe we could change the name.